Themes/default/languages/Install.english.php

```diff
@@ -167,14 +167,17 @@
 $txt['ftp_password'] = 'Password';
 $txt['ftp_password_info'] = 'The password to login with. <em>This will not be saved anywhere.</em>';
 $txt['ftp_path'] = 'Install Path';
-$txt['ftp_path_info'] = 'This is the <em>relative</em> path you use in your FTP server.';
 $txt['ftp_path_found_info'] = 'The path in the box above was automatically detected.';
 $txt['ftp_connect'] = 'Connect';
 $txt['ftp_setup_why'] = 'What is this step for?';
 $txt['ftp_setup_why_info'] = 'Some files need to be writable for SMF to work properly. This step allows you to let the installer make them writable for you. However, in some cases it won\'t work - in that case, please make the following files 777 (writable, 755 on some hosts):';
 $txt['ftp_setup_again'] = 'to test if these files are writable again.';
+$txt['ftp_login'] = 'Your FTP connection information';
+$txt['ftp_login_info'] = 'This web installer needs your FTP information in order to automate the installation for you. Please note that none of this information is saved in your installation, it is just used to setup SMF.';
+$txt['ftp_path_info'] = 'This is the <em>relative</em> path you use in your FTP client <a href="' . $_SERVER['PHP_SELF'] . '?ftphelp" onclick="window.open(this.href, \'\', \'width=450,height=250\');return false;" target="_blank" rel="noopener">(more help)</a>.';
```

Comment:
Please move any PHP variable to a sprintf() declaration wherever this txt var is used. The same for the HTML, if it can be moved away form the txt string.

Reply:
better to undo this change instead, it's taken care of here #6121 and it will probably create conflicts or unwanted results.

Reply:
#6121 has been merged which means this branch needs to either be rebase against release-2.1 or the changes been pulled from the same origin branch